homeassistant/components/system_bridge/sensor.py

```diff
@@ -252,6 +285,275 @@ def extra_state_attributes(self) -> dict[str, Any]:
         }
 
 
+class SystemBridgeGpuMemoryFreeSensor(SystemBridgeSensor):
+    """Defines a GPU memory free sensor."""
+
+    def __init__(
+        self, coordinator: SystemBridgeDataUpdateCoordinator, index: int, name: str
+    ) -> None:
+        """Initialize System Bridge sensor."""
+        super().__init__(
+            coordinator,
+            f"gpu_{index}_memory_free",
+            f"{name} Memory Free",
+            "mdi:memory",
+            None,
+            DATA_GIGABYTES,
+            True,
+        )
+        self._index = index
+
+    @property
+    def state(self) -> float | None:
+        """Return the state of the sensor."""
+        bridge: Bridge = self.coordinator.data
+        return (
+            round(bridge.graphics.controllers[self._index].memoryFree / 10 ** 3, 2)
```

Comment:
We like to avoid ternary operator expressions when they span more than one line.